docs/examples/data_transfer_objects/defining_dtos_on_layers.py

```diff
@@ -1,31 +1,45 @@
+from dataclasses import dataclass
 from typing import List
 from uuid import UUID, uuid4
 
 from litestar import Controller, delete, get, post, put
+from litestar.app import Litestar
+from litestar.dto import DataclassDTO
 
-from .models import User, UserDTO, UserReturnDTO
+
+@dataclass
+class User:
+    id: UUID
+    name: str
+
+
+UserDTO = DataclassDTO[User]
+UserReturnDTO = DataclassDTO[User]
```

Comment:
There isn't really a point in having a distinct data and return DTO if they are both the same.

There's two things we could do:

1. change the example to only use `UserDTO` and not define `return_dto` on the router (in that case `UserDTO` will apply to both data and return). We would need to change the surrounding documentation along with this.
2. configure `UserReturnDTO` so that it is different, e.g., add a `secret` field to the model and exclude it from the return data.

I think I'd prefer 2.

Reply:
perfect!